pkg/config/config.go

```diff
@@ -108,6 +113,11 @@ func (c *Config) SetConfigFromEnv() error {
 		c.Prometheus.BuildRunRampUpDurationBuckets = buildRunRampUpDurationBuckets
 	}
 
+	prometheusHistogramEnabledLabelsEnvVarValue := os.Getenv(prometheusHistogramEnabledLabelsEnvVar)
+	if prometheusHistogramEnabledLabelsEnvVarValue != "" {
+		c.Prometheus.HistogramEnabledLables = strings.Split(prometheusHistogramEnabledLabelsEnvVarValue, ",")
```

Comment:
wondering if you need the env variable validation. I think if you always do directly a:
```
c.Prometheus.HistogramEnabledLables = strings.Split(prometheusHistogramEnabledLabelsEnvVarValue, ",")
```
and the labels are not set via the env variable, you will get an empty slice, see https://golang.org/pkg/strings/#Split . Also, this will allow you to avoid doing:

```
HistogramEnabledLables:            make([]string, 0),
```
in line 68. Can you please check if this make sense?

Reply:
Yes, works. Done.